### PR TITLE
784 image tags

### PR DESCRIPTION
**Pull request by Rowansdabomb**

### CODE REVIEW

File: dispatch/api/urls.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 router.register(r'dashboard', views.DashboardViewSet, base_name='api-dashboard')
 router.register(r'integrations', views.IntegrationViewSet, base_name='api-integrations')
 router.register(r'files',views.FileViewSet, base_name='api-files')
+router.register(r'images',views.ImageViewSet, base_name='api-images')
```
Comment (JamieRL) [inline dispatch/api/urls.py]: I think this got carried over from the other branch. router already registers images url on line 17

File: dispatch/static/manager/src/js/pages/Images/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import ImagesIndexPage from './ImagesIndexPage'
+import EditImagePage from './EditImagePage'
+// import NewPersonPage from './NewPersonPage'
```
Comment (JamieRL) [inline dispatch/static/manager/src/js/pages/Images/index.js]: Comments

File: dispatch/static/manager/src/js/pages/Images/index.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+export {
+  EditImagePage as EditImage,
+  ImagesIndexPage as Index,
+  // NewPersonPage as NewPerson
```
Comment (JamieRL) [inline dispatch/static/manager/src/js/pages/Images/index.js]: comments

File: dispatch/static/manager/src/js/pages/index.js [hunk 4]
```diff
@@ -2,6 +2,7 @@ import LoginPage from './LoginPage'
 import LogoutPage from './LogoutPage'
 import DashboardPage from './DashboardPage'
 import FilesPage from './FilesPage'
+
```
Comment (JamieRL) [inline dispatch/static/manager/src/js/pages/index.js]: Extra spacing

File: dispatch/static/manager/src/js/util/redux/actions.js [hunk 5]
```diff
@@ -190,6 +190,7 @@ export class ResourceActions {
   }
 
   deleteMany(token, ids) {
+    // debugger
```
Comment (JamieRL) [inline dispatch/static/manager/src/js/util/redux/actions.js]: remove debugger comments

**Review by JamieRL — CHANGES_REQUESTED** [review]
Just a few leftover comments and some spacing. Also I think the extra router for images is still in urls.py

**Review by Rowansdabomb — COMMENTED** [review]
ok updated the comments and spacing

**Review by JamieRL — APPROVED** [review]

**Pull request merged**